lib/theme-api-client.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+const api = 'https://api.integration.zone';
+// const api = 'https://api.bigcommerce.com';
```

Comment:
Can you uncomment this line and delete the other?
Please add a parameter to overwrite this url. Something like:
```
stencil push --host https://api.integration.zone
```